man/Kernel-class.Rd

```diff
@@ -23,7 +23,7 @@ Has methods able to connect and talk to a Jupyter server.
 
 \item{\code{new_reply(msg_type, parent_msg)}}{Prepare a reply}
 
-\item{\code{send_response(msg_type, parent_msg, socket_name, content)}}{Send a response}
+\item{\code{send_response(msg_type, parent_msg, socket_name, content, metadata = NULL)}}{Send a response}
```

Comment:
Hi @janschulz . I think I made a mistake with how I understood the metadata aspect within the JMP contract, so I will remove that and we can just proceed with the store_history commit.